components/renku_data_services/crc/db.py

```diff
@@ -514,11 +512,11 @@ async def get_users(
                         )
                     return [user.dump() for user in rp.users]
                 else:
-                    stmt = select(schemas.UserORM)
+                    stmt2 = select(schemas.UserORM)
                     if keycloak_id is not None:
-                        stmt = stmt.where(schemas.UserORM.keycloak_id == keycloak_id)
-                    res = await session.execute(stmt)
-                    orms = res.scalars().all()
+                        stmt2 = stmt2.where(schemas.UserORM.keycloak_id == keycloak_id)
+                    res2 = await session.execute(stmt2)
+                    orms = res2.scalars().all()
```

Comment:
Note here: it seems that `mypy` doesn't like re-using `stmt` and `res` (thinks they are related to type `ResourcePoolORM`).